src/site/_includes/js/eligibility.js

```diff
@@ -2747,6 +2760,34 @@ function homelessPreventionSystemResult(input) {
   return program.getResult();
 }
 
+function hudVashResult(input) {
+  const program = new Program();
+  const isUnhoused = isOneOf(input.housingSituation, [
+    'vehicle',
+    'transitional',
+    'hotel',
+    'shelter',
+    'no-stable-place']);
+  const isVeteran = input.veteran;
+  const meetsDischargeReq = not(isOneOf(input.dischargeStatus, [
+    'dishonorable',
+  ]));
+  const grossLimit = MonthlyIncomeLimits.fromAnnual(
+    cnst.hudvash.ANNUAL_INCOME_LIMITS,
+    cnst.hudvash.ANNUAL_INCOME_LIMIT_ADDL_PERSON);
+  const incomeLimit = grossLimit.getLimit(input.householdSize);
+  const underIncomeLimit = le(grossIncome(input), incomeLimit);
+
+  program.addCondition(new EligCondition('Be a U.S. veteran', isVeteran));
+  program.addCondition(new EligCondition('Not be dishonorably discharged', meetsDischargeReq));
+  program.addCondition(new EligCondition('Be experiencing homelessness',
+    isUnhoused));
+  program.addCondition(new EligCondition(
```

Comment:
I think that if the housing choice voucher doesn't have this condition, it shouldn't be present here either, for consistency reasons. Maybe this can be opened as a new issue to investigate in the future.

Reply:
Sounds good.